bottlerocket/agents/src/bin/eks-resource-agent/eks_provider.rs

```diff
@@ -20,16 +20,21 @@ use resource_agent::provider::{
 use serde::{Deserialize, Serialize};
 use serde_json::Value;
 use std::env::temp_dir;
-use std::fs::write;
 use std::path::Path;
 use std::process::Command;
 use testsys_model::{Configuration, SecretName};
 
+use std::fs::File;
+use std::io::Write;
+
 /// The default region for the cluster.
 const DEFAULT_REGION: &str = "us-west-2";
 /// The default cluster version.
 const DEFAULT_VERSION: &str = "1.24";
 const TEST_CLUSTER_CONFIG_PATH: &str = "/local/eksctl_config.yaml";
+const CLUSTER_CONFIG_PATH: &str = "/local/cluster_config.yaml";
```

Comment:
Why did you create your own cluster_config.yaml file here? Do we need both this and `TEST_CLUSTER_CONFIG_PATH`?